app/main.py

```diff
@@ -1,8 +1,83 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+        self.comfort_class = self.validate_comfort_class(comfort_class)
+        self.clean_mark = self.validate_clean_mark(clean_mark)
+        self.brand = brand
+
+    def validate_comfort_class(self, comfort_class: int) -> int:
+        if 1 <= comfort_class <= 7:
+            return comfort_class
+        return 1
+
+    def validate_clean_mark(self, clean_mark: int) -> int:
+        if 1 <= clean_mark <= 10:
+            return clean_mark
+        return 1
```

Comment:
You can see that these methods look similar, so you can create a common function and use it for each attribute. Let's do it for both classes